README.md

```diff
@@ -48,6 +49,11 @@ policies:
         imperative: true
         case: lower
         invalidLastCharacters: .
+        jira:
+          required: true
```

Comment:
This might be superfluous. What do you think of dropping this? I can't see a case where if the `jira` field is set, the user _wouldn't_ want to enforce this. Maybe I am missing something?

Reply:
I was thinking of the case where you need to provide valid Jiras for features, but a change in ci stuff might not need to have it.
But I agree,  this might not be needed.

What do you think?

Reply:
I took it out ;-)